lib/cc/engine/analyzers/violation.rb

```diff
@@ -88,6 +95,16 @@ def read_up_path
           )
         end
 
+        def fingerprint
+          digest = Digest::MD5.new
+          digest << current_sexp.file
+          digest << "-"
```

Comment:
Oh weird - missed that. I think it's fine then! Whatever the old one does we should match